conf/application-json-logger.xml

```diff
@@ -5,6 +5,8 @@
         <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
     </appender>
 
+    <logger name="uk.gov" level="${logger.uk.gov:-WARN}"/>
```

Comment:
This should be changed to

<logger name="uk.gov" level="WARN"/>

as the system property approach has been superceeded.